docs/Manpage.md

```diff
@@ -442,6 +442,8 @@ message if no logs are found.
 : The Gist will be marked private and will not appear in listings but will be
   accessible with its link.
 
+### `help`
```

Comment:
@dduugg @cho-m this doesn't look quite right, perhaps a recent regression from command changes?

Reply:
Sounds likely as https://github.com/Homebrew/brew/commit/d0c1af4f9e3f87385ff1978376f4d9a7c18da028 allows `Homebrew::CLI::Parser.from_cmd_path` to load file.

May just want to add `cmd_args { hide_from_man_page! }` assuming no other regression.

Reply:
Interesting, but it maybe seems like a command worth surfacing? I had added a [description](https://github.com/Homebrew/brew/pull/16975#discussion_r1545508968), but pulled it during code review. Happy to go either way though.

Reply:
I'm also happy either way. Could you open a PR with your favourite @dduugg?

Reply:
Done in https://github.com/Homebrew/brew/pull/17045